communication/src/AsyncTlsConnection.cpp

```diff
@@ -356,7 +357,7 @@ void AsyncTlsConnection::initServerSSLContext() {
 
   // Only allow using the strongest cipher suites.
   if (!SSL_CTX_set_ciphersuites(ssl_context_.native_handle(), config_.cipherSuite.c_str())) {
-    LOG_ERROR(logger_, "Failed to set TLS cipher suite from config : " << config_.cipherSuite.c_str());
+    LOG_WARN(logger_, "Failed to set TLS cipher suite from config : " << config_.cipherSuite.c_str());
```

Comment:
Don't put a space between the text and ":", it simplifies the search.

Reply:
updated changes